src/services/motion/types.ts

```diff
@@ -7,6 +7,9 @@ export type LinearConstraint = pb.LinearConstraint.AsObject;
 export type OrientationConstraint = pb.OrientationConstraint.AsObject;
 export type CollisionSpecification = pb.CollisionSpecification.AsObject;
 export type MotionConfiguration = pb.MotionConfiguration.AsObject;
+export type GetPlanResponse = pb.GetPlanResponse.AsObject;
+export type ListPlanStatusesResponse = pb.ListPlanStatusesResponse.AsObject;
+export type PlanStateMap = pb.PlanStateMap;
```

Comment:
Instead of exporting the PlanStateMap, let's export this type:

```suggestion
type ValueOf<T> = T[keyof T];
export const { PlanState } = pb;
export type PlanState = ValueOf<typeof pb.PlanState>;
```

We do this for [another enum type](https://github.com/viamrobotics/viam-typescript-sdk/blob/c8e466076507dc31f23b205a59666f2f509581f4/src/components/encoder/encoder.ts#L6-L8). This allows library users to use `PlanState` as either enum value lookup or type of value.

For example:

```
import { PlanState } as VIAM from '@viamrobotics/sdk';

// as a value lookup
PlanState.PLAN_STATE_FAILED // => 4

// as a type
function foo(ps: PlanState) { ... } // ps must be 0,1,2,3,4 or compilation fails
```